int64 realPos1
int64 motPos1
float64 command1
int64 realPos2
int64 motPos2
float64 command2
int64 adc1
int64 adc2
int64 adc3
int64 adc4
